src/nci_util.h

```diff
@@ -76,6 +76,19 @@ nci_discovery_ntf_copy(
     const NciDiscoveryNtf* ntf)
     NCI_INTERNAL;
 
+NciModeParam*
+nci_mode_param_copy(
+    const NciModeParam* param,
+    NCI_MODE mode)
+    NCI_INTERNAL;
+
+NciActivationParam*
+nci_activation_param_copy(
+    const NciActivationParam* param,
+    NCI_RF_INTERFACE intf,
+    NCI_MODE mode)
+    NCI_INTERNAL;
```

Comment:
Wouldn't these be needed by libnciplugin? In that case they have to be in a public header. Move these prototypes to `include/nci_util.h` and rename this file `to src/nci_util_p.h` (and include `<nci_util.h>`)

Reply:
Thank, I have fixed that